spec/models/instrument_price_policy_calculations_spec.rb

```diff
@@ -313,7 +314,7 @@
       reservation.actual_end_at = reservation.reserve_end_at + 10.minutes
       new_costs = policy.calculate_cost_and_subsidy reservation
       expect(new_costs[:cost]).to eq(29.1666669.to_d)
-      expect(new_costs[:subsidy]).to eq(8.1666669.to_d)
+      expect(new_costs[:subsidy].round(2)).to eq(8.17.to_d.round(2))
```

Comment:
Can you use `Float::DIG` here?
```suggestion
      expect(new_costs[:subsidy]).to eq(8.1666669.to_d(Float::DIG))
```